docs/examples/free5gc/free5gc-single.yaml

```diff
@@ -56,6 +56,164 @@ free5gc-webui:
     service:
       type: ClusterIP
       nodePort: ""
+    configuration:
+            logger: 
+                enable: true # true or false
+                level: info # how detailed to output, value: trace, debug, info, warn, error, fatal, panic
+                reportCaller: false # enable the caller report or not, value: true or false
 
 deployUpf: false
 deployDbPython: false
+
+free5gc-smf:
```

Comment:
now, i see 2 or 4 spaces for 1 TAB in this file, please use 2 spaces only

✅ good
```
foo:
  bar: 123

bar:
  foo: 123
```
✅ good:
```
foo:
    bar: 123

bar:
    foo: 123
```

❌ bad:
```
foo:
  bar: 123

bar:
    foo: 123
```